Refactor(eos_designs)!: Network services enhance guards for OSPF (#5707)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

      ipv4_acl: IPV4
```

### `vrfs[].ospf.process_id` and `underlay_ospf_process_id` must have different values for all VRFs other than `default`

> Till AVD 5.x, It was allowed to configure the same value for `vrfs[].ospf.process_id` and `underlay_ospf_process_id` for any VRF.

Reviewer: ```suggestion
In AVD 5.x, it was possible to configure the same value for `vrfs[].ospf.process_id` and `underlay_ospf_process_id` for any VRF which was leading to potentially overwritten configuration.
```

> Starting AVD 6.0.0, setting the same value for `vrfs[].ospf.process_id` and `underlay_ospf_process_id` will not be allowed for any VRF other than `default` vrf.

Reviewer: ```suggestion
Starting AVD 6.0.0, setting the same value for `vrfs[].ospf.process_id` and `underlay_ospf_process_id` is not allowed for any VRF other than `default` vrf.
```

> An error will be raised for each vrf having the same `ospf.process_id` as `underlay_ospf_process_id`.

Reviewer: ```suggestion
An error will be raised if any VRF having the same `ospf.process_id` as `underlay_ospf_process_id`.
```

> An error will be raised if any VRF having the same `ospf.process_id` as `underlay_ospf_process_id`.

Reviewer: ```suggestion
An error is raised if any VRF have the same `ospf.process_id` as `underlay_ospf_process_id`.
```
